content/v2.0/cloud/get-started.md

```diff
@@ -18,29 +18,34 @@ _See the differences between {{< cloud-name "short">}} and InfluxDB OSS
 [below](#differences-between-influxdb-cloud-and-influxdb-oss)._
 
 ## Start for free
+
 Start using {{< cloud-name >}} at no cost with the [Free Plan](/v2.0/cloud/pricing-plans/#free-plan).
 Use it as much and as long as you like within the plan's rate-limits.
 Limits are designed to let you monitor 5-10 sensors, stacks or servers comfortably.
 Once you're ready to grow, [upgrade to the Usage-based Plan](/v2.0/cloud/account-management/upgrade-to-usage-based-plan/).
 
 ## Sign up
 
-1. Go to [InfluxDB Cloud 2.0]({{< cloud-link >}}), enter your email address and password, and click **Sign Up**.
-2. InfluxDB Cloud requires email verification to complete the sign up process. Verify your email address by opening
-   the email sent to the address you provided and clicking **Verify Your Email**.
-3. Choose your cloud provider. Currently, **AWS** is the only cloud provider available.
+1. Choose one of the following:
+   - To subscribe for a plan through InfluxData, go to [InfluxDB Cloud 2.0]({{< cloud-link >}}), enter your credentials, and then click **Sign Up**.
+   - To subscribe for a plan through AWS Marketplace, go to [Amazon Marketplace](https://aws.amazon.com/marketplace), search for **InfluxDB Cloud 2.0**, and then follow the prompts to subscribe. After you click **Set Up Your Account**, enter your credentials, and then click **Sign Up**.
+
+       {{%note%}} **Note:** You can sign up for a new InfluxDB Cloud 2.0 account through **either** InfluxData or AWS Marketplace. Currently, we do **not support** using an existing InfluxDB Cloud 2.0 account to sign up for an AWS Marketplace subscription.
```

Comment:
then we have "account" 
In line 33 note, we could swap "account" with "plan" or "subscription" in the first sentence, but not really in the second sentence.

Reply:
Accounts are different from plans/subscriptions. A user has a user account and that account uses either the Free Plan or the Usage-Based Plan. I assume the AWS Marketplace subscriptions are just Usage-Based plans?

Reply:
An AWS Marketplace Subscription is established _for_ an InfluxData Usage-Based plan, within a particular user's account

Reply:
I suspect they may feel the same in some cases because there's a 1:1 relationship between Plan and Subscription.

Reply:
"Currently, we do **not support** using an existing InfluxDB Cloud 2.0 account to sign up for an AWS Marketplace subscription" - maybe rephrase to "Currently we do not support using an existing InfluxDB Cloud 2.0 account to sign up for InfluxData via the AWS Marketplace".  I'm not sure that folks will figure out that signing up for an AWS Marketplace subscription is the same as signing up for Influx via the AWS marketplace.

Reply:
While the terms differ conceptually, the steps to create an account/plan/subscription (at least from UX point of view) are the same. Reason I bring up--for this note specifically, I wanted to point out I used "account" in the 1st and 2nd sentence, even though we could say "sign up for a new...plan" in the first sentence. Thoughts?

Reply:
I removed the first sentence in the note, given we're saying as much in Step 1.